build-compiler-rt.sh

```diff
@@ -122,6 +122,13 @@ for arch in $ARCHS; do
         -DCMAKE_CXX_FLAGS_INIT="$CFGUARD_CFLAGS" \
         $SRC_DIR
     cmake --build . ${CORES:+-j${CORES}}
+
+    # Skip install on arm64ec, we merge archives instead.
+    if [ $arch = arm64ec ]; then
```

Comment:
For comparisons like these, my shell scripting style is to enclose both the variable and the value with quotes, i.e. `if [ "$arch" = "arm64ec" ]; then`. In case the variable would happen to expand to an empty string, it makes sure the test command doesn't misparse the arguments. Same below on line 157.